Documentation/ApiOverview/Hooks/Events/Backend/AfterPageColumnsSelectedForLocalizationEvent.rst

```diff
@@ -0,0 +1,84 @@
+.. include:: ../../../../Includes.txt
+
+
+.. _AfterPageColumnsSelectedForLocalizationEvent:
+
+
+============================================
+AfterPageColumnsSelectedForLocalizationEvent
+============================================
+
+Event to listen to after the form engine has been initialized (= all data has been persisted).
+
+The PSR-14 event :php:`\TYPO3\CMS\Backend\Controller\Event\AfterPageColumnsSelectedForLocalizationEvent`
+will be dispatched after records and columns are collected in the :php`LocalizationController`.
+
+The event receives:
+
+* The default columns and columnsList built by :php:`LocalizationController`
+* The list of records that were analyzed to create the columns manifest
+* The parameters received by the :php`LocalizationController`
+
+The event allows changes to:
+
+* the columns
+* the columnsList
+
+This allows third party code to read or manipulate the "columns manifest" that gets displayed in the
+translation modal when a user has clicked the ``Translate`` button in the page module, by implementing a listener for the :php:`\TYPO3\CMS\Backend\Controller\Event\AfterPageColumnsSelectedForLocalizationEvent` event.
```

Comment:
Here I'd drop the event name. Otherwise one might feel to re-read the whole string to make sure it's the same event. Since we are in a section about this event, it's fine to refer to it as "the" event.